rectools/models/nn/bert4rec.py

```diff
@@ -82,7 +82,13 @@ def _collate_fn_train(
         self,
         batch: List[Tuple[List[int], List[float]]],
     ) -> Dict[str, torch.Tensor]:
-        """TODO"""
+        """
```

Comment:
While I was testing configs, I found that we can't do `session_max_len=session_max_len + 1` here https://github.com/MobileTeleSystems/RecTools/blob/5413b1492b620d625a4d69ca5e5d5b62eccfad08/rectools/models/nn/bert4rec.py#L248

This breaks the logic of inheritance and configs. Let's remove +1. Please do this in this PR to check that all tests pass (to avoid changing expected outputs we can just change argument while initializing the model for the test.